src/Controller/ListVMsController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+final class ListVMsController extends AbstractController
+{
+    #[Route('/list/v/ms', name: 'app_list_v_ms')]
```

Comment:
Le nom de l'endpoint doit-être /liste/vms non ?

```suggestion
    #[Route('/list/vms', name: 'app_list_vms')]
```